README.md

```diff
@@ -299,6 +299,9 @@ project(myproject LANGUAGES CXX C)
 # Set PCH to be on by default for all non-Developer Mode Builds
 set(ENABLE_PCH_USER_DEFAULT ON)
 
+# enable Conan
+set(ENABLE_CONAN_DEFAULT ON)
```

Comment:
@lefticus is this the preferred method for setting options for `dynamic_project_options`? Can't this be an argument passed to the function?

Reply:
I think ENABLE_* should be done before the include of `DynamicProjectOptions.cmake`. Because the defaults are updated before calling `dynamic_project_options()`.

Reply:
In my opinion, this is not a modular design. The defaults should be called once the function is called, not when it is included.

Reply:
Created an issue for it
https://github.com/cpp-best-practices/project_options/issues/72